libs/ui-toolkit/src/components/button/button.tsx

```diff
@@ -30,14 +30,14 @@ const primary = [
   'enabled:active:bg-vega-yellow-550 enabled:active:border-vega-yellow-550',
 ];
 const secondary = [
-  'text-white dark:text-black',
```

Comment:
ahm, are you sure this works fine on every theme?

Reply:
As per designs:

<img width="624" alt="Screenshot 2023-07-14 at 12 44 32" src="https://github.com/vegaprotocol/frontend-monorepo/assets/26136207/6b0a04e4-afa0-4ee5-86bc-c5baebb81842">
ge per theme np?

Reply:
(Secondary is pink)

Reply:
alright